torch/csrc/jit/codegen/cuda/ops/alias.cpp

```diff
@@ -94,7 +94,10 @@ TensorView* squeeze(
 }
 
 TensorView* unsqueeze(TensorView* x, int64_t dim) {
-  TORCH_INTERNAL_ASSERT(dim >= 0 && dim < x->nDims());
+  if (dim < 0) {
```

Comment:
Would we need to handle negative dim with `squeeze` as well?
https://github.com/csarofeen/pytorch/blob/85eceb82e0b0a39c8d92f869ae1b6be42198833d/torch/csrc/jit/codegen/cuda/ops/alias.cpp#L85-L91

Reply:
Ha, we should, let me update this.